novelai/novelai.py

```diff
@@ -140,28 +152,76 @@ async def novelai(self,
                       sampler_version: Optional[str],
                       noise_schedule: Optional[str],
                       decrisper: Optional[bool],
+                      model: Optional[ImageModel],
+                      reference_image1: Optional[discord.Attachment],
+                      reference_image_strength1: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image_info_extracted1: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image2: Optional[discord.Attachment],
+                      reference_image_strength2: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image_info_extracted2: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image3: Optional[discord.Attachment],
+                      reference_image_strength3: Optional[app_commands.Range[float, 0.0, 1.0]],
+                      reference_image_info_extracted3: Optional[app_commands.Range[float, 0.0, 1.0]],
                       ):
+        max_image_size = await self.config.max_image_size()
+        if reference_image1:
+            if "image" not in reference_image1.content_type or not reference_image1.width or not reference_image1.height or not (reference_image1.size / 1024 / 1024) <= max_image_size:
+                return await ctx.response.send_message(f"reference_image1 must be a valid image and less than {max_image_size}.", ephemeral=True)
+        if reference_image2:
+            if "image" not in reference_image2.content_type or not reference_image2.width or not reference_image2.height or not (reference_image2.size / 1024 / 1024) <= max_image_size:
+                return await ctx.response.send_message(f"reference_image1 must be a valid image and less than {max_image_size}.", ephemeral=True)
```

Comment:
Missing the units (MB) in the user-facing message.

Same for the other 2 checks above and below, and the 3 checks in img2img.